casepro/contacts/tests.py

```diff
@@ -32,13 +32,15 @@ def test_save(self):
             language="eng",
             is_stub=False,
             fields={'age': "34"},
-            __data__groups=[("G-001", "Customers")]
+            __data__groups=[("G-001", "Customers")],
+            urns=["phone:0821234567"],
```

Comment:
The identity store doesn't really care what the address type is, but I think we can try to stick to `tel` for casepro.